quartz/plugins/transformers/ofm.ts

```diff
@@ -109,7 +109,7 @@ const capitalize = (s: string): string => {
 // (#[^\[\]\|\#]+)? -> # then one or more non-special characters (heading link)
 // (|[^\[\]\|\#]+)? -> | then one or more non-special characters (alias)
 const wikilinkRegex = new RegExp(/!?\[\[([^\[\]\|\#]+)?(#[^\[\]\|\#]+)?(\|[^\[\]\|\#]+)?\]\]/, "g")
-const highlightRegex = new RegExp(/==(.+)==/, "g")
+const highlightRegex = new RegExp(/==([^=]*)==/, "g")
```

Comment:
```suggestion
const highlightRegex = new RegExp(/==([^=]+)==/, "g")
```

iirc you can't have an empty highlight